win32ss/user/user32/CMakeLists.txt

```diff
@@ -81,7 +81,7 @@ add_library(user32 MODULE
     ${CMAKE_CURRENT_BINARY_DIR}/user32.def)
 
 set_module_type(user32 win32dll UNICODE ENTRYPOINT DllMain 12)
-target_link_libraries(user32 user32_vista_static user32_wsprintf wine win32ksys ${PSEH_LIB})
+target_link_libraries(user32 user32_vista_only_static user32_vista_static user32_wsprintf wine win32ksys ${PSEH_LIB})
```

Comment:
why "vista only static"? These are the NT6-only (not NT 6.1+) functions?

Reply:
No i was just trying to desperate what's imported for winesyncs what what is used for actual exports since there's a LOT of code just added for vista+ apps

Reply:
ok so it was preferrable to have these in a separate lib. Makes sense.